### PR TITLE
fix(remarkable): remove async as a dependency

### LINKED ISSUE #1: remove "async" and other unwanted require statements
When `grunt-remarkable` is used without installing `async` it throws an error, so this should be declared as a dependency or devDep.
From https://github.com/rwky/grunt-remarkable/blob/master/tasks/remarkable.coffee#L4
### Error thrown:

```
$ grunt remarkable
Loading "remarkable.coffee" tasks...ERROR
>> Error: Cannot find module 'async'
Loading "remarkable.js" tasks...ERROR
>> Error: Cannot find module 'async'
```

EDIT: On close inspection, these werent required. So remove `async`.

@rwky

### PR DESCRIPTION
**Pull request by TDA**
Fixes #1 
@rwky

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,18 +4,19 @@
...
   "version": "1.0.0",
   "keywords": ["grunt","markdown","remarkable"],
-  "homepage":"https://github.com/rwky/grunt-remarkable",
+  "homepage": "https://github.com/rwky/grunt-remarkable",
   "bugs": { "url": "https://github.com/rwky/grunt-remarkable/issues" },
   "engines": {
     "node": ">=0.10"
   },
   "dependencies": {
+    "async": "2.0.0-rc.5",
```
Comment (vladikoff) [inline package.json]: @TDA can we use a stable version of async here?

Comment (pdehaan) [inline package.json]: Latest stable version of **async** seems to be "1.5.2"

https://github.com/caolan/async/releases

Comment (TDA) [inline package.json]: Done.

**Comment by rwky** [comment]
Fixed in 60e5ef13d3e0b8b03541a3064cd51a2f96073ef6

**Pull request closed without merging**